### PR TITLE
Feat display eric

### PR DESCRIPTION
**Pull request by Thiemann96**
# Added display to home sketch

  - DIsplay cycles through 5 pages. Each page displays 2 to 3 sensors from the senseBox:home Setup. If a sensor is not connected, i.e. is not defined via #define statements, instead of displaying a value, `not connected` is shown. 

 - A page will be displayed for 5 seconds

- Status messages at the start give indication of state of sensebox, e.g. when connecting to a network the ssid is shown and the current status (connected <=> not connected) is shown.

### CODE REVIEW

**Comment by felixerdy** [comment]
I implemented the display routine in the lora sketch in 8171837 but I don't have a display here so I can't test for now

**Comment by ubergesundheit** [comment]
It would be also very awesome to have this tested automatically by CI. To do this, integrate your changes in https://github.com/sensebox/node-sketch-templater/blob/master/arduino-test/build-sketches.js

File: templates/homev2_wifi.tpl [hunk 1]
```diff
@@ -435,6 +496,145 @@ void loop() {
...
+        display.setTextColor(BLACK, WHITE);
+        display.println(F("TSL&VEML"));
+        display.setTextColor(WHITE, BLACK);
+        display.println();
+        display.setTextSize(1);
+        display.print(F("Lux:"));
+#ifdef TSL45315_CONNECTED
+        display.println(TSL.readLux());
+#else
+        display.println(F("not connected")));
```
[the remark this answers is not in the document]
Comment (Thiemann96) [reply templates/homev2_wifi.tpl:546]: Good catch, will fix!

**Comment by Thiemann96** [comment]
> I implemented the display routine in the lora sketch in [8171837](https://github.com/sensebox/node-sketch-templater/commit/8171837b7f8eea3bbf89f2e7925b732188d00596) but I don't have a display here so I can't test for now

I'll have a look at it!

**Comment by Thiemann96** [comment]
> It would be also very awesome to have this tested automatically by CI. To do this, integrate your changes in https://github.com/sensebox/node-sketch-templater/blob/master/arduino-test/build-sketches.js

Is this how ?  dcd8889abb8c026d9edaa31318ef52d28c826c0c

**Comment by ubergesundheit** [comment]
> Is this how ? dcd8889

nice. Thanks

**Pull request merged**